client/data/disputes/reducer.js

```diff
@@ -11,7 +11,7 @@ import { map, keyBy } from 'lodash';
 import TYPES from './action-types';
 import { getResourceId } from 'utils/data';
 
-const defaultState = { byId: {}, queries: {} };
+const defaultState = { byId: {}, queries: {}, summary: {}, cached: {} };
```

Comment:
Do we still need `byId` here (also in tests)?

Reply:
No, `byId` is still used as the single Dispute objects are stored in there after fetching.